languages/scala/reference/exercise-concepts/hamming.md

```diff
@@ -0,0 +1,32 @@
+# Concepts of hamming
+
+[Example implementation](https://github.com/exercism/scala/blob/master/exercises/hamming/example.scala)
+
+## General
+
+- object or case class: used as class wrapper for exercies
+- functions: used as main entry point for the exercise
+- function arguments: input strands passed as arguments
+- return values: return a value from a function
+- conditionals using if: conditionally execute logic using an `if` statement
+- strings: `string` passed as the two input parameters
+- character operations: `!=`
```

Comment:
Would this be "equality comparison operations," or is there something particular in comparing characters?

Reply:
I think "equality comparison operations" would work. I suppose I was getting overly specific.